mtproto/session_store.go

```diff
@@ -47,15 +51,19 @@ func (s *SessFileStore) Save(sess *SessionInfo) (err error) {
 
 func (s *SessFileStore) Load(sess *SessionInfo) error {
 	f, err := os.Open(s.FPath)
-	if errors.Is(err, fs.ErrNotExist) {
+	if !s.RequirePresent && errors.Is(err, fs.ErrNotExist) {
 		return ErrNoSessionData.Here()
 	}
 	if err != nil {
 		return merry.Wrap(err)
```

Comment:
If `RequireReadable = true` and file exists and is unreadable (no read permission for example), the login process will fail. Is that intended? Looks unexpected to me.